microprofile.ls/com.redhat.microprofile.ls/src/main/java/com/redhat/microprofile/model/Node.java

```diff
@@ -81,7 +81,7 @@ public String getText() {
 		if (start == -1 || end == -1) {
 			return null;
 		}
-		return getOwnerModel().getText(start, end);
+		return getOwnerModel().getText(start, end).replace("\\\n", "");

```

Comment:
Could you explain which usecase you wish to cover? Is it working with Windows OS line separator `\r\n` ?

Reply:
The use case for this was for multiline property keys.

For example, for this application.properties file
```
quarkus.\
application.name = name
```
the property key text would be: `quarkus.\\napplication.name`, which would create an unknown property error because of the `\\n`. This line removes the `\\n` so that the property can be recognized.

> Is it working with Windows OS line separator \r\n ?
Thank you for reminding me. I did not test with Windows.

Reply:
Ok thank you for the clarification. getText is used for comments too with formatting, is it annoying to remove every time \n? It's just a question if you need to do that every time and just for some cases (like getPropertyName).

Reply:
@xorye I think we should have 2 methods :

 * getText(boolean skipMultiLine)
 * getText() which call getText(false)

because getText() is used too for formatting. I suspect that you should have a trouble with https://github.com/redhat-developer/quarkus-ls/blob/master/microprofile.ls/com.redhat.microprofile.ls/src/main/java/com/redhat/microprofile/services/MicroProfileFormatter.java#L155 when you will try to format property key in multi line (you will loose multi line, I think)

Separate into to methods will give the capability to use getText in different context 

 * for getPropertyName, \n must be escaped
 * for formatting it should not.

Reply:
@angelozerr I tried to do this for my latest commit. To skip multiline, I was decided to look for \\n and \\r\n.